plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/AccumuloTransactionHandle.java

```diff
@@ -13,60 +13,22 @@
  */
 package io.trino.plugin.accumulo;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
This class seems unused since you've converted it to record and no callsite is changed to use a new method name.

Reply:
Drop instead.

Reply:
No, this is used from AccumuloConnector class.